app/src/main/java/org/wikipedia/places/PlacesFragment.kt

```diff
@@ -189,6 +244,19 @@ class PlacesFragment : Fragment(), LinkPreviewDialog.Callback, MapboxMap.OnMapCl
         }
     }
 
+    private fun updateSearchCardViews() {
```

Comment:
Hi @dbrant, thanks for making the changes to the map so I can continue working on the link preview dialog.

I just noticed one issue: the compass is overlapped by the search bar view, can you fix that? 
<img src="https://github.com/wikimedia/apps-android-wikipedia/assets/2435576/2212d14e-345e-4cd0-989f-f8fdfd9ddfdf width="400" />

Reply:
good catch, and Done!